packages/react-notifications/package.json

```diff
@@ -44,7 +44,7 @@
     "moment": "2.22.2",
     "moment-timezone": "0.5.23",
     "prop-types": "15.6.2",
-    "react-redux": "5.1.1",
+    "react-redux": "6.0.0",
```

Comment:
Given this we should/have to require also to update in the mc-fe as I would assume the new/old context otherwise would not pair too well together?

Reply:
I think it would make more sense to get the update working to 6.0.0 in MC-fe before bumping this here?

Reply:
Is that even possible? The store is defined here, right? Maybe I am failing to grasp something.

Reply:
What I am concerned about is the behaviour changes around dispatching actions in constructors / componentWillMount.
https://github.com/reduxjs/react-redux/releases

Reply:
It says it’s actually an improvement because now reads are consistent. I don’t think we should have a problem with that

Reply:
Yeah, but are we sure that but we aren't relying on the existing behavior? 🤔

Reply:
Aren‘t we mixing topics again. 

1. Behavior might have changed but we are unsure if affected
2. Path to upgrade. First here or first mc-fe

To understanding 2. should be here first and then mc-fe as the consumer. Otherwise my understanding would we that we would define stores here on legacy context and consume on the updated context. That sounds wrong. Is that understanding false?
To my understaning for 1. we can not definately tell, correct. Is there a path we see to get more certainty?

Reply:
In terms of merged PRs, the path to upgrade should happen as you described, here first and then in MC-FE.

In terms of testing this PR, I think it would be worthwhile to do the old copy and paste trick, and upgrade MC-FE redux to 6.0.0, and make sure everything seems fine?

Reply:
Sure. Seems reasonable.

Reply:
So I tried to get it to work and failed. I would recommend us taking a `react-redux` update separately from any other dep for now. I removed the update in the last commit. Would the PR be fine as it stands then? /cc @montezume and @qmateub

Reply:
Could you mention what failed exactly and what problems did you encounter?

Reply:
I would like to take that discussion elsewhere on an issue and move on with this. It was basically something that store was not configured correctly.

Reply:
Alright thanks

Reply:
The upgrade to v6 is now taken care in here: #197